frontend/src/App.tsx

```diff
@@ -30,7 +32,7 @@ const App: React.FC = () => {
 
   return (
     <React.Fragment>
-      <NotificationsPortal />
+      <NotificationsPortal store={store} />
```

Comment:
You up versioned our dependency and passed the store, help me understand what this does for us (or the chrome)?

Reply:
There's a problem with shared react-redux in a couple of our components where they can't access directly the redux provider.

The version update was just so we are up to date with the config repo, there's been just a few adjustments to how files are split.

Reply:
Interesting, we may need to look at how the SDK will interact with the Chrome in the long run... but this works for now. Thanks for the explanation! 👍